src/core/lib/attribute-manager.js

```diff
@@ -513,16 +520,30 @@ export default class AttributeManager {
   // Update attribute buffers from any attributes in props
   // Detach any previously set buffers, marking all
   // Attributes for auto allocation
-  /* eslint-disable max-statements */
+  /* eslint-disable max-statements, no-continue */
   _setExternalBuffers(bufferMap) {
```

Comment:
For discussion: for now external buffers of type (luma.gl Buffer) can only be updated as part of layer props. Should we also add public API so applications can just update an attribute with a different Buffer object type when needed?